### ISSUE TITLE
The witnesses order is not correct with biginteger in Java

### ISSUE THREAD

**Issue by chenquanyu** [neow3j/neow3j#79]
The scriptHash of verificationScript is little-endian (Hash.sha256AndThenRipemd160(script))
So the BigInt value should use reverse order to get BigInt:
```
this.scripts.sort(Comparator.comparing(s -> Numeric.toBigInt(s.getScriptHash())));

this.scripts.sort(Comparator.comparing(s -> Numeric.toBigInt(ArrayUtils.reverseArray(s.getScriptHash()))));
```
![image](https://user-images.githubusercontent.com/4393234/62205179-74bb4200-b3c1-11e9-82c8-c54f28f7d469.png)

![image](https://user-images.githubusercontent.com/4393234/62198876-096b7300-b3b5-11e9-829a-e97a6b32700e.png)

**Comment by claudemiller** [comment]
This is fixed with pull-request #77.

**Comment by gsmachado** [comment]
This is published on the `2.1.0-SNAPSHOT`. It will be included on the `2.1.0` release.
